modules/navier_stokes/test/tests/finite_volume/ins/mms/skew-correction/test.py

```diff
@@ -0,0 +1,31 @@
+import mms
+import unittest
+from mooseutils import fuzzyEqual, fuzzyAbsoluteEqual
+
+def run_spatial(*args, **kwargs):
+    try:
+        kwargs['executable'] = "../../../../../../"
+        return mms.run_spatial(*args, **kwargs)
+    except:
+        kwargs['executable'] = "../../../../../../../combined/"
+        return mms.run_spatial(*args, **kwargs)
+
+class TestVortexSkewCorrected(unittest.TestCase):
+    def test(self):
+        velocity_labels = ['L2u', 'L2v']
+        pressure_labels = ['L2p']
+        labels = velocity_labels + pressure_labels
+        df1 = run_spatial('skewed-vortex.i', 5, y_pp=labels, mpi=2)
+
+        fig = mms.ConvergencePlot(xlabel='Element Size ($h$)', ylabel='$L_2$ Error')
+        fig.plot(df1, label=labels, marker='o', markersize=8, num_fitted_points=3, slope_precision=1)
+        fig.save('skewed.png')
+        for key,value in fig.label_to_slope.items():
+            print("%s, %f" % (key, value))
+            if key in velocity_labels:
+                self.assertTrue(fuzzyAbsoluteEqual(value, 2., .15))
+            else:
+                self.assertTrue(fuzzyAbsoluteEqual(value, 1., .2))
```

Comment:
Do you know why we lose an order for pressure?

Reply:
I have no idea, I haven't found good papers about this. It might have something to do with the Rhie-Chow interpolation. I like to think about it gaining 0.5 order of pressure :P.

Reply:
Haha of course. We can keep this in the back of our minds as something we'd like to understand at some point